SECURITY.md

```diff
@@ -6,6 +6,8 @@ Patches for [ **illinois-app** ] will only be applied to the following versions:
 
 | Version | Supported |
 | ------- | ------------------ |
+| 4.1.36 | :white_check_mark: |
+| 4.1.35 | :white_check_mark: |
```

Comment:
Since there is no officially released version called 4.1.35, I think it could be removed from supported versions. If it sounds okay, please remove this in the next release. Thanks.